addons/portal/static/src/js/portal_chatter.js

```diff
@@ -211,13 +242,110 @@ var PortalChatter = Widget.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onAttachmentChange: function (ev) {
+        var files = ev.target.files,
+            attachments = this.get('attachmentIDs');
+
+        if (attachments.length + files.length > 10) {
+            this.display_alert("Oops! You can not upload more than 10 files.");
+        }
+        else if (!_.isEmpty(_.filter(files, function (file) {return (file.size / 1024 / 1024) > 50;}))) {
+            this.display_alert("Oops! You can not upload the file size more than 50MB.");
```

Comment:
It should be this.display_alert("Oops! You can not upload a file larger than 50MB.");